godot/export_presets.cfg

```diff
@@ -18,10 +18,10 @@ script_export_mode=2
 
 [preset.0.options]
 
-custom_template/debug="/home/scroggo/Engines/godot/bin/web_dlink_debug.zip"
-custom_template/release="/home/scroggo/Engines/godot/bin/web_dlink_release.zip"
+custom_template/debug=""
```

Comment:
This seems to work, but I am confused why. Before I built my custom templates, I'm pretty sure Godot complained that I didn't have a template to use, though I don't remember the details.

Reply:
I would _guess_ Godot was complaining about the missing 4.3 export templates. Your custom templates were not included in the repo, and it built OK without them.